### ISSUE TITLE
vk.xml: Aliased enums using value in `<require>` don't specify extends

### ISSUE THREAD

**Issue by Dav1dde** [KhronosGroup/Vulkan-Docs#753]
Related to: https://github.com/KhronosGroup/Vulkan-Docs/issues/744

The bug that Enum's use `value` instead of `alias` has already been filed. But there is another issue with such enums:

```xml
            <require>
                <enum value="9"                                                 name="VK_EXT_DEBUG_REPORT_SPEC_VERSION"/>
                <enum value="&quot;VK_EXT_debug_report&quot;"                   name="VK_EXT_DEBUG_REPORT_EXTENSION_NAME"/>
                <enum offset="0" extends="VkStructureType"                      name="VK_STRUCTURE_TYPE_DEBUG_REPORT_CALLBACK_CREATE_INFO_EXT"/>
                <enum offset="1" extends="VkResult" dir="-"                     name="VK_ERROR_VALIDATION_FAILED_EXT"/>
                <enum value="VK_STRUCTURE_TYPE_DEBUG_REPORT_CALLBACK_CREATE_INFO_EXT" name="VK_STRUCTURE_TYPE_DEBUG_REPORT_CREATE_INFO_EXT"/>
                <enum offset="0" extends="VkObjectType"                         name="VK_OBJECT_TYPE_DEBUG_REPORT_CALLBACK_EXT"          comment="VkDebugReportCallbackEXT"/>
                <enum value="VK_DEBUG_REPORT_OBJECT_TYPE_DEBUG_REPORT_CALLBACK_EXT_EXT"         name="VK_DEBUG_REPORT_OBJECT_TYPE_DEBUG_REPORT_EXT"/>
                <type name="VkDebugReportObjectTypeEXT"/>
                <type name="VkDebugReportCallbackCreateInfoEXT"/>
                <command name="vkCreateDebugReportCallbackEXT"/>
                <command name="vkDestroyDebugReportCallbackEXT"/>
                <command name="vkDebugReportMessageEXT"/>
            </require>
```
The enum `VK_STRUCTURE_TYPE_DEBUG_REPORT_CALLBACK_CREATE_INFO_EXT` correctly specifies the `extends` attribute. The enum `VK_STRUCTURE_TYPE_DEBUG_REPORT_CREATE_INFO_EXT` which uses the previously defined enum as a value does **not** have an `extends` attribute. 

This makes it really hard to actually figure out where this enum belongs, it may very well be a global enum with the same value or a define: `#define VK_STRUCTURE_TYPE_DEBUG_REPORT_CREATE_INFO_EXT VkStructureType.VK_STRUCTURE_TYPE_DEBUG_REPORT_CALLBACK_CREATE_INFO_EXT`.

**Comment by krOoze** [comment]
> it may very well be a global enum 

It actually is.
It is a global constant which has a value of "VK_STRUCTURE_TYPE_DEBUG_REPORT_CALLBACK_CREATE_INFO_EXT".

Assuming #744 gets fixed, is anything actionable here? Do you have an example that won't go away after #747?

**Comment by Dav1dde** [comment]
Okay interesting, I'll have to through all of these and double check if they are global are not.

**Comment by Dav1dde** [comment]
But since this is intended, I am going to assume this is wanted for all of these cases and close this.

Thanks!
